CMakeLists.txt

```diff
@@ -212,6 +212,11 @@ if(ENABLE_QUICHE)
     endif()
 endif()
 
+if(ENABLE_ASAN)
```

Comment:
We should make sure this option is documented by putting the variable in the cache along with all the other ENABLE_XXX options:

`option(ENABLE_ASAN "Use address sanitizer" OFF)`